### PR TITLE
OCPBUGS#44365: Hosted control planes: Configure layer2 and BGP using metallb

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OCPBUGS-44365

Link to docs preview: 
- https://85011--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-manage/hcp-manage-bm.html#hcp-bm-ingress_hcp-manage-bm

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-bm-ingress.adoc [hunk 1]
```diff
@@ -75,47 +75,55 @@ metadata:
...
 spec:
-  protocol: layer2
-  autoAssign: false
   addresses:
     - <ingress_ip>-<ingress_ip> <2>
----
+  autoAssign: false
+----
+<1> Specify the `IPAddressPool` resource name.
+<2> Specify the IP address for your environment, for example, `192.168.122.23`.
```
Comment (dfitzmau) [inline modules/hcp-bm-ingress.adoc:95]:
```suggestion
<2> Specify the IP address for your environment. For example, `192.168.122.23`.
```

File: modules/hcp-bm-ingress.adoc [hunk 2]
```diff
@@ -75,47 +75,55 @@ metadata:
...
+
+.. Apply the configuration for the IP address pool:
```
Comment (dfitzmau) [inline modules/hcp-bm-ingress.adoc:97]:
```suggestion
.. Apply the configuration for the IP address pool by entering the following command:
```

**Review by dfitzmau — APPROVED** [review]

File: modules/hcp-bm-ingress.adoc [hunk 3]
```diff
@@ -75,47 +75,55 @@ metadata:
...
-<2> Specify the IP address for your environment, for example, `192.168.122.23`.
-<3> Specify the `BGPAdvertisement` resource name.
-+
-.. Save the file as `ipaddresspool-bgpadvertisement-config.yaml`.
-+
-.. Create the resources by entering the following command:
+<1> Specify the `L2Advertisement` resource name.
+<2> Specify the `IPAddressPool` resource name.
+
+.. Apply the configuration:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/hcp-bm-ingress.adoc]: Addressed all the suggestions. Thanks.

**Pull request merged**